osu.Game/Screens/Select/BeatmapCarousel.cs

```diff
@@ -119,7 +119,7 @@ private void loadBeatmapSets(IEnumerable<BeatmapSetInfo> beatmapSets)
         {
             CarouselRoot newRoot = new CarouselRoot(this);
 
-            newRoot.AddItems(beatmapSets.Select(s => createCarouselSet(s.Detach())).Where(g => g != null));
+            newRoot.AddItems(beatmapSets.Select(s => createCarouselSet(s.Detach())).OfType<CarouselBeatmapSet>());
```

Comment:
This is sorta subtle, but I suppose it works... (hint for other readers: `OfType<>()` internally uses the `is` operator, which doubles here as a null-check).